inferred-spans/src/main/java/io/opentelemetry/contrib/inferredspans/internal/CallTree.java

```diff
@@ -427,6 +431,7 @@ int spanify(
       @Nullable Span parentSpan,
       TraceContext parentContext,
       SpanAnchoredClock clock,
+      BiConsumer<SpanBuilder, SpanContext> normalSpanParentOverride,
```

Comment:
[minor] here I would suggest to remove the `normal` prefix from the name as it's not really clear what "normal" means in this context, to me it should be either the default value or the overridden one.